cookbook/bundles/installation.rst

```diff
@@ -39,9 +39,12 @@ Now that you know the package name, you can install it via Composer:
     $ composer require friendsofsymfony/user-bundle
 
 This will choose the best version for your project, add it to ``composer.json``
-and download the library into the ``vendor/`` directory. If you need a specific
-version, add a ``:`` and the version right after the library name (see
-`composer require`_).
+and download its code into the ``vendor/`` directory. If you need a specific
+version, include it as the second argument of the `composer require`_ command:
+
+.. code-block:: bash
+
+    $ composer require friendsofsymfony/user-bundle "~2.0@dev"
```

Comment:
This is just an example of using specific version constraints, so I think it doesn't matter to use a dev version. In fact, the example is real because it was copied from the FOSUserBundle documentation.